### ISSUE TITLE
v7.10.0 and up remove images while converting word document to PDF

### ISSUE THREAD

**Issue by JonasSchubert** [gotenberg/gotenberg#757]
Hi everyone and first all thanks for this awesome project!

we are using this project as container to convert documents on the fly (during an API call) to PDFs as not all clients can read office documents, but should handle PDFs with ease.

Some days ago we updated our container to version 7.10.1 (We also tried v7.10.0).
With these versions we encountered issues converting word documents (.docx) with images. All images in the document are gone.

Could that be an issue with the used libreoffice version in the recent versions?

Kind regards
Jonas

**Comment by gulien** [comment]
Hello @JonasSchubert 👋 

On the demo instance (7.10.1), it's working as expected with [document.docx](https://github.com/gotenberg/gotenberg/blob/main/test/testdata/libreoffice/document.docx).

Out of curiosity, does it happens with `gotenberg/gotenberg:edge` too?

**Comment by JonasSchubert** [comment]
Hi @gulien 👋 

With the test file of this project I do not have issues either.

But it looks like libreoffice has issues with our file. `edge`, `7.10.1` and `7.10.0` are removing the images. The original .docx file size is around 16MB. The PDF is around 0.6MB.

I have tried to figure out whether size is an issue and downloaded your testfile. I have added images until the document had a size of 32MB. But conversion to PDF was successful, images were kept.

I would say it is an issue with our file and libreoffice - not gotenberg itself. So feel free to close this issue. I will try to figure out, why this file is faulty.

Thanks for your quick response!

**Comment by gulien** [comment]
👍 Good luck finding the root issue! Maybe an encoding issue somehow?

**Comment by anderso** [comment]
I have this exact issue, a docx document with images that are stripped from the pdf, not all images though, and there are some other artifacts as well, but the interesting thing is that I if I go back to the last version before 7.10.0 (7.9.2), it renders correctly. I have tried various later versions but none render it correctly. I have also tried the latest version of LibreOffice on Windows and a 24.x version of on Ubuntu, they have no issues with that file, only Gotenberg after 7.9.2.

@JonasSchubert did you ever find a workaround or explanation?
